packages/@aws-cdk/aws-codebuild/lib/project.ts

```diff
@@ -721,6 +721,7 @@ export class Project extends ProjectBase {
     const ret = new Array<CfnProject.EnvironmentVariableProperty>();
     const ssmIamResources = new Array<string>();
     const secretsManagerIamResources = new Array<string>();
+    const kmsIamResources = new Array<string>();
```

Comment:
Should this be a `Set` instead? Feels like if we provide 2 Secrets from another account, we should get only a single permissions statement in our IAM Role...

Reply:
You know, there's a chance the IAM library will do this for us.

A unit test will tell you if I'm right, hint hint 😉

Reply:
Wrote a test and then needed a set 😄